FLEDGE.md

```diff
@@ -155,9 +155,19 @@ The `executionMode` attribute is optional, and may contain one of the following
    mode does not have the same limitations on what top-level sites can join or leave
    the interest group.
 
-The `ads` list contains the various ads that the interest group might show.  Each entry is an object that includes both a rendering URL and arbitrary metadata that can be used at bidding time.
+The `ads` list contains the various ads that the interest group might show.  Each entry is an object that must contain an `renderUrl` property with the URL for the ad that would be shown. An ad may also have the following optional properties:
 
-The `adComponents` field contains the various ad components (or "products") that can be used to construct ["Ads Composed of Multiple Pieces"](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#34-ads-composed-of-multiple-pieces)). Similarly to the `ads` field, each entry is an object that includes both a rendering URL and arbitrary metadata that can be used at bidding time. Thanks to `ads` and `adsComponents` being separate fields, the buyer is able to update the `ads` field via the `updateUrl` without losing `adComponents` stored in the interest group.
+ * `adRenderId`: A short identifier for this ad in this context. When this field is specified it will be sent instead of the full ad object for [B&A server auctions](https://github.com/WICG/turtledove/blob/main/FLEDGE_browser_bidding_and_auction_API.md).
```

Comment:
We need to specify a type here, e.g. string or BigInt.

Reply:
Done. It has been specified to be an 8 byte string.

Reply:
>Done. It has been specified to be an 8 byte string.

Is PAAPI UTF-8 all through?

Reply:
Hmm that's an interesting point.  JavaScript and ECMAScript strings are UTF-16 (e.g. [charCodeAt()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/charCodeAt) returns a 16-bit number).  So does an 8-byte string imply at most 4 characters?

Reply:
I think Chrome uses UTF-8 for strings internally, but that's a detail we shouldn't leak.

The intent was not to only be 4 characters, but rather to be 8 characters. Perhaps it would be better for us to define it as a `ByteString` instead of a `DOMString`. Then the IDL would perform the correct conversion and would enforce that each character's code was <= 255. That would ensure that the length of the JavaScript string and the length of the internal Chrome string was the same.

Reply:
That is correct.

Reply:
I could see IG owners perhaps using accented characters with codes > 255 in interest group names. 
If there is not a warning about this constraint would you please add one in the docs?